script/test

```diff
@@ -60,18 +58,34 @@ readonly OT_OPTIONS=(
     "-DOT_MAC_FILTER=ON"
 )
 
+# List of boards to test
+readonly BOARDS=(
```

Comment:
Can we add 4164a and 4181a to the list as these are boards that SQA uses to test and which out Jenkins job currently adds in?